### DISCUSSION TITLE
mini-sectors

### DISCUSSION THREAD

**Discussion by w-i-z-a-r-d** [theOehrly/Fast-F1#170] [Ideas]
is it possible to add to the API the minisectors that are in the app, so that we would be able to retrieve the exact values without having the trouble to do comparison in delta-time?

**Comment by siddharth-tewari** [comment]
You might check out this or the updated 2022 version on my account. 
1) https://github.com/theOllieS/F1_Minisectors
2) https://github.com/siddharth-tewari/F1_Minisectors

**Reply by w-i-z-a-r-d** [reply]
i have found another way, where each coordinate X,Y given from the telemetry is considered a minisector with the time also received. by doing calculation to find the delta time of this sector I would be able to compare the delta sectors for each driver and get which driver was faster in that sector (for two drivers
<img width="1565" alt="Screen Shot 2022-05-02 at 1 53 40 PM" src="https://user-images.githubusercontent.com/68217542/166316807-f651f431-f92e-436c-aaf3-3e8e1390f5c8.png">
)

**Comment by theOehrly** [comment]
> is it possible to add to the API the minisectors that are in the app, so that we would be able to retrieve the exact values without having the trouble to do comparison in delta-time?

The problem is that there are no exact timestamp values for the mini sectors. The app only shows slower, personal best, best overall marked in yellow, green and purple, right? The API seems to provide only that information and no timestamps as far as I can tell. That information is not sufficient to compare arbitrary laps.
